app/Http/Controllers/API/AdminController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\AdminRequest;
+use App\Http\Resources\AdminResource;
+use App\Models\Admin;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AdminController extends Controller
+{
+    public function registerAdmin(AdminRequest $request)
+    {
+        //        dd($request->all());
+        $admin_data=$request->validated();
+//        dd($admin_data);
+        $adminDetails=Admin::create($admin_data);
+        return response()->json([
+            'message'=>'admin registred successfully',
+            'admin'=> new AdminResource($adminDetails)
+        ],Response::HTTP_CREATED); //added status
+    }
+
+
+    public function loginAdmin(AdminRequest $request){
+        $adminDetails=Admin::where('email',$request->email)->first();
+//        dd($adminDetails);
+        if (!$adminDetails){
+            return response()->json([
+                'message'=>'admin Not Found',
+            ],Response::HTTP_NOT_FOUND);
+        }
+
+       if(!password_verify($request->password,$adminDetails->password))
```

Comment:
@Babita3331 Please have a look at how to manually authenticate users, to avoid writing long lines of code